build_binaries.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -xeu
+
+export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+export ZEPHYR_BASE=$(pwd)/zephyr
+export ZEPHYR_SDK_INSTALL_DIR=$(pwd)/zephyr-sdk
+
+mkdir artifacts
+pushd zephyr
+
+west build -p auto -b nrf52840dk_nrf52840 samples/hello_world -- -G'Unix Makefiles'
+cp build/zephyr/zephyr.elf ../artifacts/zephyr-hello_world.elf
+
+west build -p auto -b nrf52840dk_nrf52840 samples/subsys/shell/shell_module -- -G'Unix Makefiles'
```

Comment:
I'd recommend building to two separate directories, not a single overwritten `build`

Reply:
Now all the samples are built in their respective directories.